google-cloud-spanner/src/main/java/com/google/cloud/spanner/Partition.java

```diff
@@ -92,39 +92,39 @@ static Partition createQueryPartition(
     return new Partition(partitionToken, partitionOptions, statement, queryOption);
   }
 
-  ByteString getPartitionToken() {
+  public ByteString getPartitionToken() {
     return partitionToken;
   }
 
-  String getTable() {
+  public String getTable() {
     return table;
   }
 
-  KeySet getKeys() {
+  public KeySet getKeys() {
     return keys;
   }
 
-  Iterable<String> getColumns() {
```

Comment:
We should make this return a copy or something immutable (or otherwise make sure that the `Iterable` that we are returning is always immutable). Otherwise an external caller could by accident change the partition by changing this iterable.

Reply:
Makes sense. I'll skip changing it to public for now. Will make the changes on a need basis at a later point.